datasetinsights/datasets/groceries_real.py

```diff
@@ -126,19 +110,13 @@ def __init__(
             )
         self.split = split
 
-        valid_versions = tuple(self.GROCERIES_REAL_DATASET_TABLES.keys())
-        if version not in valid_versions:
-            raise ValueError(
-                f"A valid dataset version should be set. "
-                f"Available versions are: {valid_versions}"
-            )
         logger.info(
             f"Using split {split} and version {version} for groceries real "
             "dataset."
         )
 
         self.version = version
-        self.root = os.path.join(data_root, self.LOCAL_PATH)
+        self.root = data_root
```

Comment:
I think it makes sense to rename this to data_path or data_directory or something along those lines.